src/integ_test_resources/ios/sdk/integration/cdk/custom_resources/iot_custom_authorizerUserPass_function/iot_custom_authorizerUserPass.py

```diff
@@ -0,0 +1,36 @@
+import json
+import os
+
+
+def handler(event, __):
+    print(f"### handler: {event}")
+    token = event["token"].lower()
+    effect = "Allow" if token == "allow" and event["protocolData"]["mqtt"]["username"].startswith("aws-sdk-user?SDK=iOS&Version=") else "Deny"
```

Comment:
As noted in [the Xcode PR](https://github.com/aws-amplify/aws-sdk-ios/pull/2875), since this lambda (and by extension this package) is the source of truth for what's considered a "valid" username/password combination, let's update the config values with the expected username & password and read them in the test code.

Reply:
also assert the password is present and correctly formatted, since that's part of the contracted behavior

Reply:
Updated, resolving.